### PR TITLE
Copy bib files in data folder for zipped gh-paged

### LINKED ISSUE #121: echemdb.data.cv.database.Database() does not collect bibliography
The bibliography is not downloaded when the database is built from gh-pages. The following example returns an empty bibliography

```python
from echemdb.data.cv.database import Database
db = Database()
db.bibliography
```

This is because the BIB files are not copied from `literature` folder into the corresponding folders in `website/generated/svgdigitizer` when the JSON and CSV files are created during the digitization process. The downloaded zip file does not contain any BIB files.

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Fixes #121 

- [x] Write tests
- [x] change website.generator.database (folder)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ jobs:
           pushd literature
           ls ./**/*.yaml | parallel --bar --halt now,fail=1 'svgdigitizer cv --sampling-interval .001 --package --metadata {} {.}.svg --outdir=../data/generated/svgdigitizer/{//}'
           popd
+      - name: copy literature
+        shell: bash -l {0}
+        run: |
+          pushd literature
+          find . -name '*.bib' | cpio -pdm ../data/citations
+          popd
```
Comment (DunklesArchipel) [inline .github/workflows/build.yml:34]: With this approach, the bib files are copied with the directory tree.
I guess this is not necessary and a list with bib files would be sufficient after all.

I couldn't figure out how to do it.

Comment (DunklesArchipel) [reply .github/workflows/build.yml:34]: Maybe the bib files should be stored in a `citation` folder not located in `data`, but I think that all data generated for the website should be stored in a single place. Then the folder could be called `resources` with a `data` folder and a `citation` folder.

**Comment by DunklesArchipel** [comment]
All required checks are ok

File: .github/workflows/build.yml (hunk 1 above)
Comment (linuxrider) [reply .github/workflows/build.yml:34]: Is a separation necessary?

Comment (linuxrider) [reply .github/workflows/build.yml:34]:
```
data/
     sato_2006_effect_725/
          sato_2006_effect_725.bib
          sato_2006_effect_725_f4a_solid.csv
          sato_2006_effect_725_f4a_solid.json
          sato_2006_effect_725_f4b_solid.csv
          sato_2006_effect_725_f4b_solid.json
```
This structure seems to be ok. What do you think?

Comment (DunklesArchipel) [reply .github/workflows/build.yml:34]: Yes, you are right. It is better if we put the bib file in the folder with the files. 
In the case that there is at some point another folder in `data` other than `digitized/svgdigitizer` we might have to check for duplicates.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -26,6 +26,12 @@ jobs:
           pushd literature
           ls ./**/*.yaml | parallel --bar --halt now,fail=1 'svgdigitizer cv --sampling-interval .001 --package --metadata {} {.}.svg --outdir=../data/generated/svgdigitizer/{//}'
           popd
+      - name: copy literature
+        shell: bash -l {0}
+        run: |
+          pushd literature
+          find . -name '*.bib' | cpio -pdm ../data/citations
```
Comment (DunklesArchipel) [inline .github/workflows/build.yml:33]:
```suggestion
          find . -name '*.bib' | cpio -pdm ../data/generated/svgdigitizer
```

File: echemdb/data/cv/database.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def __init__(self, data_packages=None, bibliography=None):
             data_packages = echemdb.data.remote.collect_datapackages(os.path.join('website-gh-pages', 'data', 'generated', 'svgdigitizer'))
 
             if bibliography is None:
-                bibliography = echemdb.data.remote.collect_bibliography(os.path.join('website-gh-pages', 'Literature'))
+                bibliography = echemdb.data.remote.collect_bibliography(os.path.join('website-gh-pages', 'data', 'citations'))
```
Comment (DunklesArchipel) [inline echemdb/data/cv/database.py:70]:
```suggestion
                bibliography = echemdb.data.remote.collect_bibliography(os.path.join('website-gh-pages', 'data', 'generated'))
```

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # Digitized data
 /data/generated
+/data/citations
```
Comment (DunklesArchipel) [inline .gitignore:3]:
```suggestion
```

File: echemdb/website/generator/database.py [hunk 5]
```diff
@@ -39,7 +39,8 @@
         '..',
         '..',
         '..',
-        'literature'))
+        'data',
+        'citations'))
```
Comment (DunklesArchipel) [inline echemdb/website/generator/database.py:43]:
```suggestion
        'data'))
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (DunklesArchipel) [reply .github/workflows/build.yml]: See also further discussion on folder structures in #123

**Review by linuxrider — APPROVED** [review]

**Pull request merged**